pantalaimon/store.py

```diff
@@ -60,6 +60,19 @@ def to_content(self, content: Dict, mime_type: str) -> Dict[Any, Any]:
 
         return content
 
+    def to_thumbnail(self, content: Dict, mime_type: str) -> Dict[Any, Any]:
```

Comment:
It's a bit weird that this takes the content, modifies it and returns it. Maybe just remove the return.

Reply:
Makes sense. I did that as a direct copy of the MediaInfo.to_content() method, I'll remove the return from both of them(?)

Reply:
Sure.

Reply:
Thanks, that's done in the latest commit!